content/_help/specific-agencies/ssa._fr.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: help
+title: SSA.gov
+category: specific-agencies
+permalink: /fr/help/specific-agencies/ssa/
+order: 3
+---
+Login.gov ne peut répondre qu'aux questions liées au processus de connexion et à la création d'un compte sur Login.gov. Votre compte Login.gov n'a aucune incidence et ne contient aucune information sur [votre carte/numéro de sécurité sociale](https://www.ssa.gov/) ou vos prestations sociales.
+
+Veuillez ne pas envoyer à Login.gov de données sensibles vous concernant ou de numéros d'identification.
+
+Login.gov ne peut répondre qu'aux questions liées au processus de connexion et à la création d'un compte sur Login.gov.
+
+Si vous possédez déjà un compte sur [SSA.gov](https://www.ssa.gov/), vous n'avez pas besoin de créer un compte sur Login.gov, sauf si la SSA l'exige.
+
+Veuillez contacter [SSA.gov](https://www.ssa.gov/) directement si vous avez des questions concernant:
+
+* Votre numéro de sécurité sociale
+* Le remplacement de votre carte de sécurité sociale
+* Le signalement d'un numéro de sécurité sociale volé
+* D'autres informations liées aux prestations (page FAQ de la SSA: [https://faq.ssa.gov/en-US/](https://faq.ssa.gov/en-US/))
+* La vérification de vos droits en matière de:
+    * Retraite
+    * Pension d'invalidité
+    * Revenu de sécurité supplémentaire (SSI)
+    * Assurance maladie
+
+Appelez le 1-800-772-1213 entre 8 h et 19 h du lundi au vendredi ou appelez le numéro de l'ATME au 1-800-325-0778. Ou soumettez votre [demande via le formulaire Web](https://secure.ssa.gov/emailus/).
+
+Si vous vivez à l'étranger, SSA.gov fournit [les coordonnées par pays](https://www.ssa.gov/foreign/)
```

Comment:
```suggestion
Si vous vivez à l'étranger, SSA.gov fournit [les coordonnées par pays](https://www.ssa.gov/foreign/).
```

Reply:
Good catch. They ended with colons in the word doc but I forgot to finish fixing on the other versions.